src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -338,12 +338,24 @@ export default class OrganizationDiscover extends React.Component {
             />
           )}
           {shouldRenderEditMode && (
-            <QueryEdit
+            <QueryFields
               queryBuilder={queryBuilder}
               isFetchingQuery={isFetchingQuery}
               onUpdateField={this.updateField}
-              onRunQuery={this.runQuery}
-              reset={this.reset}
+              actions={[
```

Comment:
I think if this was a React `node`, it'd allow for greater flexibility in the future (e.g. what if you needed something that wasn't a button, or different alignment). It'd be a bit more verbose with all of the components, but I think it'd be more clear since you know what's going to be rendered without having to dig too deep into where `actions` is used.

Reply:
Yeah, good point